src/tentaclio/__init__.py

```diff
@@ -15,26 +15,27 @@
 from .api import *  # noqa
 from .protocols import *  # noqa
 from .urls import *  # noqa
+from .registries import get_stream_handler_registry
 
 
 # Local files
-URL.register_handler("", LocalFileHandler())
-URL.register_handler("file", LocalFileHandler())
+get_stream_handler_registry().register("", LocalFileHandler())
```

Comment:
Would it be more concise to call the factory/singleton constructor once and assign it to a variable, even though it's the same object? - also see comment below

Reply:
I'll do that